src/app/portlets/content-types/layout/content-types-layout.component.spec.ts

```diff
@@ -211,10 +215,55 @@ describe('ContentTypesLayoutComponent', () => {
             });
 
             fixture.detectChanges();
-            const pTabPanel = this.contentType.queryAll(By.css('p-tabPanel'))[2];
+            const pTabPanel = this.contentType.queryAll(By.css('p-tabPanel'))[3];
             const iframe = pTabPanel.query(By.css('dot-iframe'));
 
             expect('/html/content_types/push_history.jsp?contentTypeId=2&popup=true').toBe(iframe.componentInstance.src);
         });
     });
+
+    describe('Relationship tab', () => {
+        beforeEach(() => {
+            this.contentType = fixture.debugElement.query(By.css('.content-type'));
+            comp.contentTypeId = '1';
+
+            fixture.detectChanges();
+
+            this.relationshipTab = this.contentType.queryAll(By.css('p-tabPanel'))[1];
```

Comment:
This is not a good way to select the "Relationship tab" because if we change it of position this will break, you should use an ID or a BEM class.